UM/Signal.py

```diff
@@ -121,6 +134,14 @@ def getType(self):
     #   function will be called on the next application event loop tick.
     @call_if_enabled(_traceEmit, _isTraceEnabled())
     def emit(self, *args, **kwargs):
+        FlameProfiler.updateProfileConfig()
```

Comment:
Here also, I would greatly prefer it if we can have a switch to disable all this behaviour. Considering the amount of calls to this method, I am really hesitant to add any sort of overhead to it.